fabric-object-builders/src/main/java/net/fabricmc/fabric/api/block/FabricBlockSettings.java

```diff
@@ -43,56 +39,70 @@
  * FabricBlockSettings.create() and add .build() at the end to return the
  * vanilla Block.Settings instance beneath.
  */
-public class FabricBlockSettings {
+public final class FabricBlockSettings {
```

Comment:
Do we want this? Why?

Reply:
I must've done this absentmindedly out of habit, I can de-finalize it if you wish people to extend the implementation

Reply:
Actually... Yes, I will have to de-finalize, this is technically breaking binary compatibility.

Reply:
No, no, no. Why prevent other users from expanding on FabricBlockSettings themselves?

Reply:
Sorry I meant yes as in I will have to de-finalize it. It had no final modifier before this pull request and I cannot change that if I am to maintain binary compatibility.

Reply:
That is *not* what I meant. I was asking about your rationale in favor of finalizing it, as not finalizing it was my explicit decision.

Reply:
I saw it as a utility class... I also realise I need to move the constructors and the delegate field back to being protected, oops